konig-core/src/main/java/io/konig/core/vocab/DCL.java

```diff
@@ -0,0 +1,33 @@
+package io.konig.core.vocab;
+
+/*
+ * #%L
+ * Konig Core
+ * %%
+ * Copyright (C) 2015 - 2018 Gregory McFall
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+
+import org.openrdf.model.URI;
+import org.openrdf.model.impl.URIImpl;
+
+public class DCL {
```

Comment:
Please remove DCL from src/main.  This is a specific kind of security classification that is relevant only to Pearson and has no place in the konig open source project which is designed to be completely generic.

Where it is referenced in test cases, you should just create the URI directly in the test case (or move the DCL vocabulary to src/test/java).